test/logformats_test.py

```diff
@@ -393,6 +395,21 @@ def _setup_instance(self):
         )
 
 
+class TestMF4FileFormat(ReaderWriterTest):
```

Comment:
You may wrap this in `try: from can import MF4Writer, MF4Writer [the actual "class TestMF4FileFormat ..."]; except ImportError: pass`